spec/models/conference_spec.rb

```diff
@@ -778,6 +778,33 @@
     end
   end
 
+  describe '#scheduled_event_distribution' do
+    let!(:conference) { create(:conference) }
```

Comment:
>Just did not want to create conference in all the tests.

@Ana06 sorry by create here I mean call. Basically I was thinking that we should not call conference variable in both our test instead it should created immediately for both our test that's why I was using `let!` instead of `let` for conference. Since conference is needed in mostly all tests so even if we decide to add more test for scheduled_event_distribution we won't have to call conference variable in the new test.